moveit_ros/warehouse/README.md

```diff
@@ -0,0 +1,156 @@
+# MoveIt! ROS Warehouse
+
+This package provides methods to access the MoveIt! warehouse. 
+
+
+## Export to text
+
+An executable is available to allow to export the scene and queries that are available in the Moveit! warehouse to text files:
+
+```
+rosrun moveit_ros_warehouse moveit_warehouse_save_as_text
+```
+
+It has different parameters that can be specified:
+
+* `help`: Show help message
+* `host`: Host for the DB. Default 127.0.0.1.
+* `port`: Port for the DB. Default 33829.
+* `output_directory`: Directory to save the generated files
+* `scene`: Saves the scenes available in the warehouse. It generates .scene files. If it is not specified the queries are saved.
+* `cartesian`: Save queries in cartesian space (start and end pose of eef). If not specified they will be saved in joint space by default.
+* `eef`: Specify the end effector (Only needed when the cartesian option is set). Default: last link. 
+* `group_prefix`: Specify the group prefix you'd like to plan with. This is useful if you want to save the queries only for the arm and it has a prefix, so only those joints will be considered (e.g. "ra")
```

Comment:
I don't really get the role of this prefix. Is this a joint-name prefix? Or is it a MoveIt group?
In MoveIt, queries are always tied to a specific JointModelGroup, and this group should be saved together with the query, which I am missing from your text files. If you import queries, how can MoveIt know, to which group to apply this query?